core/src/trezor/ui/layouts/tt_v2/webauthn.py

```diff
@@ -13,8 +19,39 @@ async def confirm_webauthn(
     info: ConfirmInfo,
     pageable: Pageable | None = None,
 ) -> bool:
-    raise NotImplementedError
+    if pageable is not None:
+        # TODO: how to even replicate this situation?
```

Comment:
A bit tricky, first apply this patch (will try to turn this into PR):
```patch
diff --git a/vendor/fido2-tests/tests/vendor/trezor/utils.py b/vendor/fido2-tests/tests/vendor/trezor/utils.py
index fa4ba3c..8339b57 100644
--- a/vendor/fido2-tests/tests/vendor/trezor/utils.py
+++ b/vendor/fido2-tests/tests/vendor/trezor/utils.py
@@ -9,11 +9,12 @@ from trezorlib.transport import enumerate_devices, get_transport
 
 
 def get_device():
+    interact = os.environ.get("INTERACT") == "1"
     path = os.environ.get("TREZOR_PATH")
     if path:
         try:
             transport = get_transport(path)
-            return TrezorClientDebugLink(transport)
+            return TrezorClientDebugLink(transport, auto_interact=not interact)
         except Exception as e:
             raise RuntimeError("Failed to open debuglink for {}".format(path)) from e
 
@@ -21,7 +22,7 @@ def get_device():
         devices = enumerate_devices()
         for device in devices:
             try:
-                return TrezorClientDebugLink(device)
+                return TrezorClientDebugLink(device, auto_interact=not interact)
             except Exception:
                 pass
         else:
```

Then start an emulator and run `INTERACT=1 make test_emu_fido2 TESTOPTS="-v -k multiple_rk"`.

The important part here is to create a layout that accepts list of (app_name, account_name) and looks like what you trezorui2.confirm_webauthn except it allows you to choose the account name by swiping, and needs to return an index of the account along with CONFIRMED/CANCELLED. I suppose we can worry about integrating it with `fido2.py` later.

Reply:
trezor/fido2-tests#2